code/modules/surgery/organs/organ_external.dm

```diff
@@ -269,9 +268,9 @@
 	//If limb took enough damage, try to cut or tear it off
 	if(owner)
 		if(sharp && !(limb_flags & CANNOT_DISMEMBER))
-			if(brute_dam >= max_damage && prob(brute / 2))
+			if(brute_dam >= max_damage && owner.health <= HEALTH_THRESHOLD_CRIT && prob(brute))
```

Comment:
This change isn't documented in the PR, and effectively means you can *only* delimb people with sharp weapons if they're in crit first. That's...kind of a big change, and if it was intentional, please update the PR body to reflect it.